uberon_edit.obo

```diff
@@ -195227,6 +195227,27 @@ relationship: dc-creator http://orcid.org/0000-0003-2034-601X
 relationship: part_of UBERON:0004647 ! liver lobule
 property_value: http://purl.org/dc/elements/1.1/date 2020-10-05T15:52:22Z xsd:dateTime
 
+[Term]
+id: UBERON:8410000
+name: duodeno-jejunal junction
+def: "The anatomical junction between the duodenum and the jejunum." []
+synonym: "duodeno-jejunal flexure" EXACT []
+synonym: "duodenojejunal flexure" EXACT []
+synonym: "duodenojejunal junction" EXACT []
+synonym: "flexura duodenojejunalis" EXACT [] {http://purl.obolibrary.org/obo/uberon/core#LATIN=""}
```

Comment:
I think EXACT is fine, but that LATIN should be a synonym_type.  Did you check other examples in Protege?

Reply:
@dosumis 

> I think EXACT is fine, but that LATIN should be a synonym_type. Did you check other examples in Protege?

I see two annotation properties, "LATIN" and "latin term" (only the latter is a subclass of synonym_type_property), and there are no annotation property usage examples for either (except for my term duodeno-jejunal junction). That's weird as I _think_ I've seen at least one of those properties used. I'll change the AP to "latin term" as a synonym_type_property, and I'll add to the agenda for the next CL/Uberon meeting.

Reply:
Yep - should be 'latin term' in Protege see
![image](https://user-images.githubusercontent.com/112839/104635568-f7a98880-5699-11eb-8aef-6def83e58595.png) - but this will show up as LATIN in OBO.

Reply:
There probably needs to be some clean up of cl-edit.owl to remove incorrect term - but we need to be careful:  Protege bug means usage is not shown.

Reply:
> There probably needs to be some clean up of cl-edit.owl to remove incorrect term - but we need to be careful: Protege bug means usage is not shown.

As discussed at the CL editors meeting today, this will be addressed in https://github.com/obophenotype/uberon/issues/1727. Note, the duplication is in Uberon but not CL, so no action required there.